pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/performance/xml/UseStringBufferForStringAppends.xml

```diff
@@ -6,15 +6,15 @@
 
     <test-code>
         <description>failure case</description>
-        <expected-problems>1</expected-problems>
-        <expected-linenumbers>7</expected-linenumbers>
+        <expected-problems>2</expected-problems>
+        <expected-linenumbers>6,7</expected-linenumbers>
```

Comment:
I'm not sure we should report on each concatenation. There may be many concatenations on the same variable and I'm sure this can be very noisy. Maybe it would be nicer to flag the variable declaration, and add the line number of the first concatenation to the message. Our regression tester only flags artificial pieces of code though... Let's come back to this later (I'll add a note in the [backlog](https://github.com/pmd/pmd/projects/3#card-58283473))